drivers/pinmux/stm32/pinmux_stm32f4.h

```diff
@@ -15,10 +15,14 @@
 #define STM32F4_PINMUX_FUNC_PA0_PWM2_CH1	STM32_PINMUX_FUNC_ALT_1
 #define STM32F4_PINMUX_FUNC_PA0_UART4_TX	STM32_PINMUX_FUNC_ALT_8
 
+#define STM32F4_PINMUX_FUNC_PA1_ETH		STM32_PINMUX_FUNC_ALT_11
```

Comment:
For this file, can you group and sort per pin (then sort per increasing order of ALT) so it's a bit easier to read and check for inconsistencies/duplicates?